portality/models/suggestion.py

```diff
@@ -142,8 +142,19 @@ def set_suggester(self, name, email):
             return
         self._set_suggestion_property("suggester", sugg)
 
-    def save(self, **kwargs):
+    def _sync_owner_to_journal(self):
+        if self.current_journal is None:
+            return
+        from portality.models import Journal
+        cj = Journal.pull(self.current_journal)
+        if cj is not None and cj.owner != self.owner:
+            cj.set_owner(self.owner)
+            cj.save(sync_owner=False)
+
+    def save(self, sync_owner=True, **kwargs):
         self.prep()
+        if sync_owner:
```

Comment:
I didn't think of the endless loop that would otherwise occur when I first read the issue / PR :). Nice.